evidence/general-mechanics/bugs.md

```diff
@@ -53,6 +53,21 @@ Better understanding of Abyss bugs/issues for cleaner runs.
 
 **Significance:** This is significant as some people rely on pure RNG when trying to get that "perfect run". For example, using Xingqiu's Q and following up with two E's \(when using Sacrificial Sword\) to generate as much energy recharge as possible for the next Q is often used in vape comps. When attempting the "perfect run", people may restart several times at the first few seconds of starting a floor to get the Sacrificial Sword's passive to activate. However, they will notice that Xingqiu's E has a 21 second CD which does not get reset upon retrying a stage, forcing them to retry the floor AGAIN. This is especially noticeable on characters with long CDs on elemental skills or bursts.
 
+## Domains
+
+**By:** Puffin\#9920
+**Added:** 02/26/2022
+
```

Comment:
The discussion line needs to be here, and without the space between the ] and the (